### DISCUSSION TITLE
How to get  projects from subgroups？

### DISCUSSION THREAD

**Discussion by ayunwSky** [python-gitlab/python-gitlab#2533] [General]
I can use `projects = group.projects.list()` to get projects from group with python-gitlab library,but how can I get  projects from subgroups？

**Comment by nejch** [comment]
@ayunwSky for specific arguments you should always check the upstream API documentation - https://docs.gitlab.com/ee/api/groups.html#list-a-groups-projects.

You're looking for this:

```python
projects = group.projects.list(include_subgroups=True)
```
